dev/ci/cases/gfsv17/marine3dvar.yaml

```diff
@@ -17,10 +17,10 @@ base:
 
 prepoceanobs:
   use_exp_obs: "YES"
-  dmpdir_exp: /scratch3/NCEPDEV/da/common/
+  dmpdir_exp: "${BASE_DATA}/experimental_obs"
```

Comment:
Add a comment at the end that this will not work for real time or the retros until we stage all the obs in glopara, which will be a while.